catalog/app/containers/NavBar/Help.js

```diff
@@ -247,11 +238,17 @@ function DocsExternalLink() {
   )
 }
 
-export default function Help({ className, onQuery, ...props }) {
+function normalizeSyntaxItem(s) {
+  const multilineToLine = s.replace(/\s/g, '')
+  const trailingSpace = ' '
```

Comment:
i dont think it makes sense to instantiate this variable every time the function gets called, just `return "{mulitilineToLine} "` (wrong quotes bc of markdown syntax) or even make it into a oneliner

Reply:
The performance cost is insignificant here in my opinion, the transpiller even can remove these variables. But I think it's simpler to read code when all things have some explanations through variable names.

Reply:
For example, in the future when I will see `{multiline} ` I can think that this extra space is a mistyping

Reply:
Converted to oneliner